File: C#/zadanie1.cs
Console.WriteLine("Zadanie 1");

Console.Write("Podaj pierwszą liczbę: ");
double liczba1 = Convert.ToDouble(Console.ReadLine());

Console.Write("Podaj drugą liczbę: ");
double liczba2 = Convert.ToDouble(Console.ReadLine());

Console.Write("Podaj operację (+, -, *, /): ");
string operacja = Console.ReadLine();

if (operacja == "+")
{
    Console.WriteLine("Wynik: " + (liczba1 + liczba2));
}
else if (operacja == "-")
{
    Console.WriteLine("Wynik: " + (liczba1 - liczba2));
}
else if (operacja == "*")
{
    Console.WriteLine("Wynik: " + (liczba1 * liczba2));
}
else if (operacja == "/")
{
    if (liczba2 != 0)
    {
        Console.WriteLine("Wynik: " + (liczba1 / liczba2));
    }
    else
    {
        Console.WriteLine("Błąd: Nie można dzielić przez zero!");
    }
}
else
{
    Console.WriteLine("Nieznana operacja!");
}

File: Python/zadanie1.py
print("Zadanie 1")

liczba1 = float(input("Podaj pierwszą liczbę: "))
liczba2 = float(input("Podaj drugą liczbę: "))
operacja = input("Podaj operację (+, -, *, /): ")

if operacja == "+":
    wynik = liczba1 + liczba2
    print(f"Wynik: {wynik}")
elif operacja == "-":
    wynik = liczba1 - liczba2
    print(f"Wynik: {wynik}")
elif operacja == "*":
    wynik = liczba1 * liczba2
    print(f"Wynik: {wynik}")

elif operacja == "/":
    if liczba2 != 0:
        wynik = liczba1 / liczba2
        print(f"Wynik: {wynik}")
    else:
        print("Błąd: Nie można dzielić przez zero!")
else:
    print("Nieznana operacja!")